xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js

```diff
@@ -353,10 +353,26 @@ XWiki.widgets.LiveTable = Class.create({
       if (descriptor.type === 'hidden') {
         return;
       }
-      // The column's display name to be used when displaying the reponsive version.
+      // The column's display name to be used when displaying the responsive version.
       var displayName = descriptor.displayName || column;
       var fieldName = column.replace(/^doc\./, 'doc_');
-      if (column === '_actions') {
+      // When a cell is part of a non-viewable row (except for the actions that simply stays empty), we display a 
+      // message indicating that the content is not present because the current user does not have the right to view it.
+      if (!row['doc_viewable'] && !row[fieldName]) {
+        const notViewableCellMessage = "$services.localization.render('livetable.cell.emptyValue')";
```

Comment:
```suggestion
        const notViewableCellMessage = $jsontool.serialize($services.localization.render('livetable.cell.emptyValue'));
```

In order to make sure the (injected) translation message doesn't break the JavaScript code.